concepts/05 Widgets/Scheduler/00 Getting Started with Scheduler/15 Edit Appointments.md

```diff
@@ -0,0 +1,74 @@
+Users can add, update, and remove appointments. To configure these operations, use the [editing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/) object. 
```

Comment:
Specify how users can add, update, and remove appointments (edit form, dragging, resizing) and whether it all is enabled by default.